swingset/src/main/java/com/nqadmin/swingset/models/OptionMappingSwingModel.java

```diff
@@ -66,7 +66,7 @@
  * @param <O2>  option2 type; if present, supplementary display string
  * 
  * @see AbstractComboBoxListSwingModel for general description of using this class
- * @see <a target="_top" href="https://javadoc.io/doc/com.glazedlists/glazedlists/latest/ca/odell/glazedlists/swing/AutoCompleteSupport.html">GlazedLists AutoCompletion javadoc</a>
+ * @see <a href="https://javadoc.io/doc/com.glazedlists/glazedlists/latest/ca/odell/glazedlists/swing/AutoCompleteSupport.html">GlazedLists AutoCompletion javadoc</a>
```

Comment:
Why? Did you try it and see the difference?

Reply:
Few issues because I can't view internal and external link in the same window at work. However I see one is trying to bring up the external pages in the right frame vs clearing the window. Found that target= is fine if it's after the href.

However, while researching, if target=_blank then it will try a new tab which is nicer in my opinion so giving that a try.

Reply:
target="_blank" rel="noopener noreferrer" to prevent 'tabnapping'

Reply:
Cool. I hadn't heard of tabnapping.